src/golang/cmd/executor/executor/workflow.go

```diff
@@ -38,8 +38,9 @@ func NewWorkflowExecutor(spec *job.WorkflowSpec, base *BaseExecutor) (*WorkflowE
 		return nil, err
 	}
 
-	engineReaders := GetEngineReaders(base.Readers)
-	engineWriters := GetEngineWriters(base.Writers)
+	engineReaders := getEngineReaders(base.Readers)
```

Comment:
To confirm: will`getEngineRepos` be replacing `getEngineReaders` and `getEngineWriters` once it is completely merged in?

Reply:
Yup thats the plan